package.json

```diff
@@ -98,5 +97,8 @@
       "build-js"
     ]
   },
-  "snyk": true
+  "snyk": true,
+  "devDependencies": {
```

Comment:
any reason this is in dev dependencies and not in dependencies as the others? — think it may need to be in dependencies to make it work properly on federalist

Reply:
So this was just from running the same command as I did in the `uswds` repo: `npm i --save-dev chrome-remote-interface@latest`.

Reply:
In `uswds`, it was also saved to dev dependencies.

Reply:
yeah, I _think_ this one needs to be different because of how the deps are loaded in federalist. I think it needs to be in regular deps and not dev deps

Reply:
It looks like Federalist built properly with no errors: https://federalist-proxy.app.cloud.gov/preview/uswds/uswds-site/update-chrome-remote/ Is there another reason to change it besides this?

Reply:
Not really — but we have all the others in dependencies. It is not a problem to have this one alone in devDependencies, but because it's different, it makes it seem as though it has some unique characteristic that the others do not. It makes you ask _why_ this one is here and the others are not. If there's a good reason to make it different, it should be different. If not, it should not.

Reply:
I've also asked @toolness and #federalist-support bc I'm not confident that I know the answer.

Reply:
@thisisdano Isn't the reason it's a dev dependency on the previous repo is to make sure Snyk doesn't scan it? We were getting a lot of non-relevant errors on `uswds` repo which is why Brian divided them into dependencies and dev dependencies to make sure only the real dependencies in production are scanned.

Reply:
honestly, I think every dependency we have is a dev dependency, since all of it is in the service of testing, compilation, transpilation, etc, and none of it is required for the "app" to run

Reply:
i had though the only reason we included stuff in dependencies was in some way related to the federalist build process — but that may not be the case

Reply:
@thisisdano we can also just make everything a  regular dependency now and make a note to switch everything out to a dev dependency later on. How does that sounds?

Reply:
yeah — I _do_ think that everything should probably just be a devDependency (in here and in `uswds`, too), but it'll take a little bit of testing to make sure it works properly when we do that

Reply:
I'm just gonna go ahead and pre-approve this.

Reply:
Updated in 61489b9.